membership-attribute-service/app/services/ScanamoBehaviourService.scala

```diff
@@ -15,8 +15,8 @@ class ScanamoBehaviourService(client: AmazonDynamoDBAsyncClient, table: String)
   def run[T] = ScanamoAsync.exec[T](client) _
 
   override def get(userId: String): Future[Option[Behaviour]] =
-    run(scanamo.get('userId -> userId).map(_.flatMap(_.toOption)))
-
+    run(scanamo.get('userId -> userId).map(_.flatMap(_.right.toOption)))
```

Comment:
would be nice to know what the (new?) left represents - if it's a message we could at least leftMap it into the logger so at least it's not lost completely